packages/genesys-spark-tokens/data/ui/forms_and_inputs.json

```diff
@@ -1634,7 +1634,7 @@
       },
       "text": {
         "gap": {
-          "value": "{spacing.4xs}",
+          "value": "{spacing.xs}",
```

Comment:
@dohajnal I think this change affects the selector card component, is that meant to be there?

Reply:
@conor-darcy , I think this one should not be there, but I will check with the team and get back. There were some changes merged that I pulled into my branch as well.

Reply:
@conor-darcy , checked w the team, this change is most likely a mistake. Should I change it or is it easier that you discard this change?

Reply:
@dohajnal Probably easier for you to do it on Figma. At least we'll know its gone from both places immediately.

Reply:
@conor-darcy , I just checked this one in Tokens Studio, and it looks like I have the correct value on the branch (pls see screenshot)
![image](https://github.com/MyPureCloud/genesys-spark/assets/141759283/83bdd6e1-0fae-46f6-b1a6-1f2e79b523f5)